person.rb

```diff
@@ -0,0 +1,22 @@
+# Class Person file.
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    true if of_age? || @parent_permission
+  end
+
+  private
+
+  # Naming/PredicateName: Rename is_of_age? to of_age? Rubocop error, that's why I have changed the name
+  def of_age?
+    true if @age >= 18
+  end
```

Comment:
- [ ] Same here as I have mentioned above your method here is not complete because you are not handling the `falsy` case and it will return `nil` if it is a `falsy` case (for example if the age is equal to 17). Let's fix that following the example above.